jvm/gradle/libs.versions.toml

```diff
@@ -14,6 +14,8 @@ material = "1.11.0"
 cargo-ndk = "0.3.4"
 compose-bom = "2024.02.00"
 kover = "0.7.6"
+datastore = "1.1.1"
+coroutines = "1.8.0"
```

Comment:
maybe we should go to 1.9.0 and also update our wallet with 2.0.0 kotlin?

i know it's not the most important thing at the moment, so it's not a requirement from my side.